scripts/sda-admin

```diff
@@ -0,0 +1,448 @@
+#!/bin/sh
+
+set -u -e
+
+myself=$0
+
+# Default values for global options.
+mq_credentials=test:test	# --mq-credentials user:pass
+mq_host=localhost		# --mq-host hostname
```

Comment:
If `mq_host` is the entire URL i.e. `http(s)://hostname:port` then you don't have to modify the script for a semi-production environment where TLS is enabled.

Reply:
> If `mq_host` is the entire URL i.e. `http(s)://hostname:port` then you don't have to modify the script for a semi-production environment where TLS is enabled.

Should I also rename the variable `mq_url` or similar?

Reply:
Yes, since this script will be used by 3:rd party people we need to make it as clear as possible what things are/mean.